easybuild/easyconfigs/r/RE2/RE2-2024-03-01-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = "CMakeMake"
+
+name = 'RE2'
+version = '2024-03-01'
```

Comment:
is there a particular reason to avoid the latest release ([2024-07-02](https://github.com/google/re2/releases/tag/2024-07-02))?

Reply:
No, I must have missed that there was a newer version, I had like a dozen deps to string up.

Reply:
OK, dealt with.